samples/bluetooth/iso_broadcast/sample.yaml

```diff
@@ -7,3 +7,10 @@ tests:
     integration_platforms:
       - qemu_cortex_m3
     tags: bluetooth
+  sample.bluetooth.iso_broadcast.bt_ll_sw_split:
+    harness: bluetooth
+    platform_allow: qemu_cortex_m3 qemu_x86 nrf52_bsim nrf52dk_nrf52832
+    integration_platforms:
+      - nrf52dk_nrf52832
+    extra_args: OVERLAY_CONFIG=overlay-bt_ll_sw_split.conf
```

Comment:
The number of boards that had a specific config before does not match the platforms/boards that use the new overlay. Is this intentional?